boards/microduino-corerf/doc.txt

```diff
@@ -0,0 +1,80 @@
+/**
+ * @defgroup    boards_microduino-corerf Microduino CoreRF
+ * @ingroup     boards
+ * @brief       Support for the Microduino CoreRF board
+
+# Hardware
+## Pinout
+
+![corerf-pinout](https://wiki.microduinoinc.com/images/d/df/RF%E5%BC%95%E8%84%9A.jpg)
+
+## Board
+The board is just a breakout for the ATmega128RFA1 MCU.
+
+## MCU Details
+| MCU            | ATmega128RFA1         |
+|:------------- -|:--------------------- |
+| Family	 | ATmega                |
+| Vendor	 | Atmel	         |
+| Package        | QFN/MLF               |
+| SRAM		 | 16Kb                  |
+| Flash		 | 128Kb                 |
+| EEPROM         | 4K                    |
+| Core Frequency | 8MHz (16MHz no power save mode) |
+| Oscillators    | 32.768 kHz & 16 MHz   |
+| Timers	 | 6 ( 2x8bit & 4x16bit )|
+|Analog Comparator| 1                    |
+| ADCs		 | 1x 15 channel 6 to 12-bit |
+| USARTs         | 2                     |
+| SPIs		 | 3 (1 SPI & 2 USART SPI) |
+| I2Cs		 | 1 (called TWI)	 |
+| Vcc		 | 1.8V - 3.6V		 |
+| Datasheet / Reference Manual | [Datasheet and Reference Manual](http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-8266-MCU_Wireless-ATmega128RFA1_Datasheet.pdf) |
+| Board Manual	 | [Wiki Page](https://wiki.microduinoinc.com/Microduino-Module_CoreRF) |
```

Comment:
I now this is super bike-shedding and jerky to ask for, but I somehow find unaligned table columns super irritating. Would you mind to align the columns? (No need to align the right column of the last two rows with the long links.)